README.md

```diff
@@ -1,53 +1,103 @@
-# {{ NAME }}
+# TimeSpan
 
-{{ DESCRIPTION }}
+A PowerShell module for formatting TimeSpan objects into human-readable strings with customizable precision and formatting options.
```

Comment:
Generalize this description to say its to manage and build functionality on top the TimeSpan object.